lib/Components/core.css

```diff
@@ -104,6 +104,7 @@
 
     --COLOR-TEXT-BLACK: #000000;
     --COLOR-TEXT-GREY: #999999;
+    --COLOR-BS-GREY:rgba(134, 134, 134,1);
```

Comment:
Identation - It should be rgba(124, 124, 124, 1). Space before 1 should be removed